Menus/RadialMenu/Scripts/RadialMenuSlot.cs

```diff
@@ -349,44 +414,108 @@ void SetIconPressed()
 
 		IEnumerator IconHighlightAnimatedShow(bool pressed = false)
 		{
-			var currentPosition = m_IconTransform.localPosition;
+			var currentPosition = m_IconContainer.localPosition;
 			var targetPosition = pressed == false ? m_IconHighlightedLocalPosition : m_IconPressedLocalPosition; // Raise up for highlight; lower for press
 			var transitionAmount = Time.unscaledDeltaTime;
 			var transitionAddMultiplier = pressed == false ? 14 : 18; // Faster transition in for standard highlight; slower for pressed highlight
 			while (transitionAmount < 1)
 			{
-				m_IconTransform.localPosition = Vector3.Lerp(currentPosition, targetPosition, transitionAmount);
+				m_IconContainer.localPosition = Vector3.Lerp(currentPosition, targetPosition, transitionAmount);
 				transitionAmount = transitionAmount + Time.unscaledDeltaTime * transitionAddMultiplier * 2;
 				yield return null;
 			}
 
-			m_IconTransform.localPosition = targetPosition;
+			m_IconContainer.localPosition = targetPosition;
 			m_IconHighlightCoroutine = null;
 		}
 
 		IEnumerator IconEndHighlight()
 		{
-			var currentPosition = m_IconTransform.localPosition;
+			var currentPosition = m_IconContainer.localPosition;
 			var transitionAmount = 1f; // this should account for the magnitude difference between the highlightedYPositionOffset, and the current magnitude difference between the local Y and the original Y
 			var transitionSubtractMultiplier = 5f;
 			while (transitionAmount > 0)
 			{
-				m_IconTransform.localPosition = Vector3.Lerp(m_OriginalIconLocalPosition, currentPosition, transitionAmount);
+				m_IconContainer.localPosition = Vector3.Lerp(m_OriginalIconLocalPosition, currentPosition, transitionAmount);
 				transitionAmount -= Time.unscaledDeltaTime * transitionSubtractMultiplier;
 				yield return null;
 			}
 
-			m_IconTransform.localPosition = m_OriginalIconLocalPosition;
+			m_IconContainer.localPosition = m_OriginalIconLocalPosition;
 			m_IconHighlightCoroutine = null;
 		}
 
+		IEnumerator AnimateSemiTransparent(bool makeSemiTransparent)
+		{
+			if (m_InsetRevealCoroutine != null)
+			{
+				// In case semiTransparency is triggered immedlately upon showing the radial menu
+				this.StopCoroutine(ref m_InsetRevealCoroutine);
+				m_CanvasGroup.alpha = 1f;
+				ShowCleanup();
+			}
+
+			const float k_FasterMotionMultiplier = 2f;
```

Comment:
Don't use k_ for local variables. You're free to use k (no underscore or simply camel case naming with no prefix. Please fix up any other areas where you have done this.